### PR TITLE
Ran clang-tidy before 9.0.x branching

### PR DESCRIPTION
**Pull request by bryancall**

### CODE REVIEW

File: .clang-tidy [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Checks: -*,modernize-use-default-member-init,modernize-loop-convert,modernize-use-bool-literals,modernize-deprecated-headers,performance-unnecessary-value-param,performance-faster-string-find,modernize-raw-string-literal,modernize-redundant-void-arg,modernize-use-nullptr,modernize-use-override
+Checks: -*,google-readability-braces-around-statements,google-readability-casting,modernize-use-using,modernize-loop-convert,modernize-use-bool-literals,modernize-deprecated-headers,performance-unnecessary-value-param,performance-faster-string-find,modernize-raw-string-literal,modernize-redundant-void-arg,modernize-use-nullptr,modernize-use-override,performance-unnecessary-copy-initialization
```
Comment (masaori335) [inline .clang-tidy]: `modernize-make-shared` & `modernize-make-unique` are also candidates.

**Comment by bryancall** [comment]
> As you may already know, changes against files for QUIC don't make sense at all.
> It changes `quicNetVCAllocator` to `netVCAllocator `, `QUICPacketHandler ` to `QUICPacketNumber `, etc.

I will built with QUIC support and fix the issues.

**Comment by bryancall** [comment]
I fixed the QUIC issues by reverting most of the QUIC files.

**Comment by randall** [comment]
[approve ci autest]

**Pull request merged**